cypress/Shared/TestCasesPage.ts

```diff
@@ -182,12 +182,24 @@ export class TestCasesPage {
 
         this.editTestCaseAddJSON(testCaseJson)
     }
+    public static enterErroneousJson(err_TestCaseJson: string): void {
 
-    public static editTestCaseAddJSON (testCaseJson:string)  :void{
+        Utilities.waitForElementVisible(TestCasesPage.aceEditor, 37700)
+        cy.wait(2000)
```

Comment:
will need to look at this too. Did you look into waiting for any particular api calls?